packages/apollo-server-testing/src/createTestClient.ts

```diff
@@ -9,7 +11,12 @@ type Mutation = { mutation: StringOrAst; query?: undefined };
 
 export default (server: ApolloServerBase) => {
   const executeOperation = server.executeOperation.bind(server);
-  const test = ({ query, mutation, ...args }: Query | Mutation) => {
+  const test = ({
+    query,
+    mutation,
+    ...args
+  }: (Query | Mutation) &
+    Omit<Parameters<typeof server.executeOperation>[0], 'query'>) => {
```

Comment:
Alternatively, we could just add a variables prop to `Query` and `Mutation` since that is the most likely to be used

Reply:
I think we should expand the use cases here as necessary but we should start with `variables` and `operationName`.

Eventually, we can consider `http` and `extensions` as they become necessary, but one of the reasons `executeOperation` came into existence was to avoid the need to have an actual HTTP server instantiated to run the operation.

For what it's worth, I know this works and avoids directly importing `GraphQLRequest` from the `requestPipelineAPI` project (which would require adding it a TypeScript project reference to `apollo-server-testing`'s `tsconfig.json`, but I can't help wish this was more easy to read!  Anyhow, I won't block on that, but if you have any thoughts on improving the readability, let me know!

Reply:
The import issue is why I did it this way, and no, I don't like it very much either 😛 . I think adding `variables` and `operationName` props to `Query` and `Mutation` would be a simpler approach, even though this is technically more correct, it doesn't actually add any value, per your comment about `http` and `extensions`

Reply:
@feinoujc Do you mind making that change?  I think we should go with that, for now.

Reply:
👍 done